### ISSUE TITLE
true type siji

### ISSUE THREAD

**Issue by fauno** [gstk/siji#8]
hi, since i couldn't mix bitmap and ttf fonts on i3status, i took the pita to convert siji to ttf :P

https://share.riseup.net/#4o90fxqDmaJ0YL_CUV8Rcw

i used a mix of https://github.com/ali1234/bitmap2ttf, manual editing and fontforge to create it.  if you're interested i can post the sources :)

**Comment by Dr-Terrible** [comment]
Yeah, PCF is outdated as a format, and doesn't play well with modern WM on Wayland. Moving to a scaling font-set is way better.

**Comment by diffficult** [comment]
@fauno could you post it again, the file is not available anymore and it would come in handy having this in .ttf file format.

**Comment by fauno** [comment]
hey, i uploaded it to [my fork](https://github.com/fauno/siji) but since a year has passed i completely forgot the procedure :P 

maybe i have the files somewhere...

**Comment by diffficult** [comment]
Thanks man, it didn't cross my mind to check your fork. If it isn't much trouble, it would be nice to see what you did. I had trouble trying to properly convert other bdf fonts to True Type. It's easier to render a valid Windows .fon file but you can only get as result only alphabet, numbers and some diacriticals, not the full font with all the special characters.
